Source/utils/display.cpp

```diff
@@ -34,6 +34,9 @@ extern SDL_Surface *renderer_texture_surface; /** defined in dx.cpp */
 Uint16 gnScreenWidth;
 Uint16 gnScreenHeight;
 Uint16 gnViewportHeight;
+Rectangle MainPanel;
+Rectangle LeftPanel;
+Rectangle RightPanel;

```

Comment:
This also needs to live in `control.cpp` :)

Reply:
It make sense :smile: